templates/repo/clone_script.tmpl

```diff
@@ -19,5 +21,8 @@
 		for (const el of document.getElementsByClassName('js-clone-url')) {
 			el[el.nodeName === 'INPUT' ? 'value' : 'textContent'] = link;
 		}
+		for (const el of document.getElementsByClassName('js-clone-url-vsc')) {
+			el['href'] = 'vscode://vscode.git/clone?url=' + encodeURIComponent(link);
```

Comment:
The linter wants to see this:
```suggestion
			el['href'] = `vscode://vscode.git/clone?url=${encodeURIComponent(link)}`;
```

Reply:
Should we revert https://github.com/go-gitea/gitea/pull/20557/commits/fae64bf57e6795203fa2bd8b0e2621b04efcde2f then?